### DISCUSSION TITLE
Não Consigo Abrir o Site

### DISCUSSION THREAD

**Discussion by Mertijis** [YuriAlessandro/gerador-ficha-tormenta20#382] [Perguntas e Respostas]
Opa, Boa tarde. Tenho tentado acessar o Site (https://fichasdenimb.com.br/). ja tentei em vários aplicativos mas não consigo de modo algum. (Opera Gx, Microsoft Edge Chrome)
![imagem_2024-11-19_185139948](https://github.com/user-attachments/assets/7ecd4a98-1f4b-4ab0-bf8f-e1f1bed99285)

**Comment by YuriAlessandro** [comment]
Tenho a impressão que isso é um problema de rede no seu PC. Aparentemente tudo está normal aqui para mim.

Você já testou no 3G/4G/5G ou em outra rede de internet?
